projects/04-burger-queen/README.md

```diff
@@ -490,3 +491,40 @@ yarn start
 * [Qué es Serverless? | FooBar - YouTube](https://www.youtube.com/watch?v=_SYHUpLi-2U)
 * [Firebase](https://firebase.google.com/)
 * [Serverless Architectures - Martin Fowler](https://www.martinfowler.com/articles/serverless.html)
+
+## 7. Funcionalidades para reforzar OA de promesas
+
+Te sugerimos implementar las siguientes funcionalidades para que
+puedas reforzar aún más los objetivos de aprendizaje de promesas.
+
+* Agrega la opción de actualizar 2 o más pedidos que esten en la
+cocina. Muestra al usuario un **único** mensaje de confirmación
+cuando **todos** los pedidos hayan sido actualizados con éxito.
+Recuerda, firestore al
+[actualizar un documento](https://firebase.google.com/docs/firestore/manage-data/add-data#update-data)
+retorna una promesa. ¿Cómo podrías mostrar el mensaje de confirmación 
+unicamente cuando todas las promesas se hayan resuelto? Te sugerimos 
+revisar la función 
+[Promise.all](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all).
+* Cuando un pedido finalice se debe hacer una petición HTTP a la url
+[https://salty-escarpment-74924.herokuapp.com/](https://salty-escarpment-74924.herokuapp.com/)
```

Comment:
Hay que editar eso entonces y incluir el funcion que responde un status https://github.com/Laboratoria/bootcamp/pull/1158#discussion_r892966656 en otro archivo

Reply:
@unjust @mfdebian @ssinuco Sergio, te recomiendo que ese webservice del descuento lo despliegues en otro servicio gratuito como [Glitch](https://glitch.com/), dado que Heroku terminará su servicio gratuito [en 3 semanas](https://help.heroku.com/RSBRUH58/removal-of-heroku-free-product-plans-faq)

Reply:
gracias @santiaguf !! al final optaremos por dejar una función que entregue un número aleatorio de manera asíncrona en forma de promesa, para que ninguna coach tenga que estar manteniendo algún servicio con respecto a eso, muchas gracias nuevamente!